test/images/agnhost/net/main.go

```diff
@@ -49,8 +50,33 @@ type logOutput struct {
 	b bytes.Buffer
 }
 
-func main() {
-	initFlags()
+// CmdNet is used by agnhost Cobra.
+var CmdNet = &cobra.Command{
+	Use:   "net",
+	Short: "TBA",
+	Long:  "TBA",
+	Args:  cobra.MaximumNArgs(2),
+	Run:   main,
+}
+
+func init() {
+	legalRunners := ""
+	for k := range runners {
+		legalRunners += " " + k
+	}
```

Comment:
nit, `strings.Join(runners, " ")`?

Reply:
It seems we can't. runners is a map, not an array of strings, which Join expects.

Reply:
Ah, I just didnt notice that. Thanks. 👍